core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -277,10 +263,12 @@ fun DropdownTextField(
     modifier: Modifier = Modifier,
     title: String,
     items: List<String>,
-    selected: (String) -> Unit
+    selected: (String) -> Unit,
+    value: String = "",
+    onSelectedOption: (String) -> Unit,
+    newSelectedOption: (String) -> Unit
```

Comment:
Do you really need two high order functions? Can I know why did you created each one please?

Reply:
update, let me know if this is a best way